guides/common/modules/proc_increasing-host-limit-for-pull-based-rex-transport.adoc

```diff
@@ -3,32 +3,28 @@
 [id="Increasing_host_limit_for_pull_based_REX_transport_{context}"]
 = Increasing host limit for pull-based REX transport
 
-You can tune the `mosquitto` MQTT server and increase the number of hosts connected to it.
+You can tune the `mosquitto` MQTT server and increase the maximum number of hosts connected to it (by default it's 1014).
 
 .Procedure
-. Enable pull-based remote execution on your {ProjectServer} or {SmartProxyServer}:
+This example configures the `mosquitto` service on a {ProjectServer} or {SmartProxyServer} to handle 5000 content hosts.
+
+. Add the following to `/etc/foreman-installer/custom-hiera.yaml` to increase the default value:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
+cat >>/etc/foreman-installer/custom-hiera.yaml <<EOF
```

Comment:
Typically we don't tell users to run shell commands to add content to files. It's very easy to mess up. In this case they can type `cat > /etc/foreman-installer/custom-hiera.yaml` and wipe out any contents they had before.